mk/support/pkg/curl.sh

```diff
@@ -96,7 +102,7 @@ pkg_link-flags () {
             -lidn)    out `pkg link-flags libidn idn` ;;
             -lssl)    out_openssl ssl ;;
             -lcrypto) out_openssl crypto ;;
-            -ldl)     dl_libs=-ldl;; # Linking may fail if -ldl isn't last
```

Comment:
Whoops, indeed. Forgot to remove it. On FreeBSD was causing a linker error, so i had to comment it out. Tried to cleanup the modifications, but someone always escape to the checks 👍